### ISSUE TITLE
Bug: Fetching from non-default branch fails

### ISSUE THREAD

**Issue by aGuyNamedJonas** [aGuyNamedJonas/startmeup#4]
![image](https://user-images.githubusercontent.com/1662929/120760707-b7498a80-c514-11eb-9358-25393deb6033.png)

**Comment by aGuyNamedJonas** [comment]
Could reproduce it with the following: 
```
npx startmeup github.com/facebook/react:portals fixtures/art testfolder

Error: Command failed with exit code 128: git clone --depth=1 https://github.com/facebook/react:portals.git /var/folders/70/4mjk6vbx59bgq08mnqczsd1h0000gn/T/startmeup-3Hkgwy
Cloning into '/var/folders/70/4mjk6vbx59bgq08mnqczsd1h0000gn/T/startmeup-3Hkgwy'...
fatal: unable to access 'https://github.com/facebook/react:portals.git/': The requested URL returned error: 400
```

**Comment by jpeeck-spring** [comment]
Resolved by #16
